samples/app-cache-meetings/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-caching-in-meetings-nodejs",
+    "source": "officeDev",
+    "title": "App Caching in Meetings",
+    "shortDescription": "This sample feature is to improve the subsequent loading time of an App",
```

Comment:
fix wording
this sample shows how to improve....

same in line 9 as well

Reply:
Fixed